herald_rules/validate_description_is_present.json

```
@@ -1,5 +1,6 @@
 {
   "name": "The template has a Description",
+  "targetURL": "https://github.com/gagoar/use-herald-action/wiki/Pr-Validations#validate-description-title-is-present",
```

```suggestion
  "targetURL": "https://github.com/gagoar/use-herald-action/wiki/PR-Validations#validate-description-title-is-present",
```
I have already changed the respective wiki page